tests/common/Touch.Unit/Touch.Client/Runner/Options.cs

```diff
@@ -0,0 +1,1111 @@
+//
```

Comment:
We can use the `Mono.Options` NuGet instead of importing this file.

Reply:
Done